content/applications/sales/crm/acquire_leads/convert.rst

```diff
@@ -2,66 +2,121 @@
-the same email address. When merging opportunities, Odoo merges the
-information into the opportunity which was created first, giving
-priority to the information present on the first opportunity.
+Then, select a :guilabel:`Salesperson` and a :guilabel:`Sales Team` to which the opportunity should
+be assigned.
 
-No information is lost: data from the other opportunity is logged in the
-chatter and the information fields for easy access.
+If the lead has already been assigned to a salesperson or a team, these fields automatically
+populate with that information.
 
-.. image:: convert/convert_04.png
+.. image:: convert/convert-leads-conversion-action.png
     :align: center
+    :alt: Create opportunity pop-up.
+
+Under the :guilabel:`Customer` heading, choose from the following options:
+
+- :guilabel:`Create a new customer`: Choose this option to use the information in the lead to create
+  a new customer.
+- :guilabel:`Link to an existing customer`: Choose this option, then select a customer from the
+  resulting drop-down menu, to link this opportunity to the existing customer record.
+- :guilabel:`Do not link to a customer`: Choose this option to convert the lead, but not link it to
+  a new or existing customer.
+
+Lastly, when all configurations are complete, click :guilabel:`Create Opportunity`.
+
+.. _sales/crm/acquire_leads/merge-leads:
 
-Would you find a duplicate yourself, ...you can also merge opportunities
-or leads even if the system doesn't propose it.
+Merge leads and opportunities
+=============================
 
-Here’s how, from the list view. Select the opportunities or leads you
-want to merge and the action button will appear. Then, you can select merge.
+Odoo automatically detects similar leads and opportunities by comparing the email addresses of the
+contact names. If a similar lead/opportunity is found, a :guilabel:`Similar Lead` smart button is
+added to the top of the lead/opportunity record.
 
-.. image:: convert/convert_05.png
+.. image:: convert/convert-leads-similar-leads.png
     :align: center
+    :alt: Similar leads smart button.
+
+To compare the details of the similar leads/opportunities, click the :guilabel:`Similar Leads`
+button. This opens a kanban view with only the similar leads/opportunities. Click into each card
+to view the details for each lead/opportunity, and confirm if they should be merged.
+
+.. important::
+   When merging, Odoo gives priority to whichever lead/opportunity was created in the system first,
+   merging the information into the first created lead/opportunity. However, if a lead and an
+   opportunity are being merged, the resulting record is referred to as an opportunity, regardless
+   of which record was created first.
+
+Once it's confirmed the leads/opportunities should be merged, return to the kanban view using
+breadcrumbs, or by clicking the smart button. Click the :guilabel:`☰ (three vertical lines)` icon
```

Comment:
Does the smart button has a name within the UI? It might be good to include it in a guilabel.